rainbowsaddle/__init__.py

```diff
@@ -12,6 +12,16 @@
 import functools
 import traceback
 
+enable_statsd = True
+stats = None
+
+try:
+    import statsd
+    stats = statsd.StatsClient(prefix='rainbowsaddle')
+except ImportError:
+    print('Statsd import failed')
```

Comment:
Want to:

- Print to stderr
- Mention whats going to happen that it failed (statsd will be disabled)